app/controllers/api/v0/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V0::BaseController < ApplicationController
+  include OpenidConnect::ProtectedResourceEndpoint
+
+  def user
+    current_token ? current_token.authorization.user : nil
+  end
```

Comment:
Does this override devise's `current_user` for all controllers that extend from Api::V0::BaseController?